modules/ossm-smcp-prod.adoc

```diff
@@ -42,6 +42,8 @@ spec:
           resources: {}
 ----
 
+.. Configure the sampling rate for production. For more information, see the Performance and scalability section.
+
```

Comment:
I know this is existing text, but I think you need a `+` between[ the _Verify the results._ steps and the command](https://github.com/openshift/openshift-docs/pull/32981/files#diff-4588b2ad0407c53d2acfa89719b3f18d6949c08e2684b962f3ae8827230043ecR49-R53).

Reply:
fixed